### ISSUE TITLE
Logged on! - But not online in Steam?

### ISSUE THREAD

**Issue by Marow** [bulgakovk/steamBot#4]
Hi,

is it normal, that the Bot is online in (Console) but in Friendlist he is not online?

**Comment by bulgakovk** [comment]
>is it normal, that the Bot is online in (Console) but in Friendlist he is not online?

Yep, It's normal. If you want to set "Online" status you can use [this method](https://github.com/DoctorMcKay/node-steam-user#setpersonastate-name)

**Comment by datamann10** [comment]
Here's the specific area you need to add this to: http://prntscr.com/9dugkf
using this code: client.setPersona(SteamUser.Steam.EPersonaState.Online);
